.circleci/config.yml

```diff
@@ -219,22 +219,7 @@ commands:
           - venv-<< parameters.py-version >>-*
 
 jobs:
-  checkout:
-    executor: docker
-    steps:
-      - checkout-and-persist
-
   prepare-system-linux:
```

Comment:
@ulope merging the jobs `checkout`, `prepare-system`, and `prepare-python` would remove the need for the egg_info generation, correct?

The workspace would be saved by only one job.